Pokedex/MainScreen.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ViewController.swift
+//  Pokedex
+//
+//  Created by Стефан  Миливоевич on 5.10.22..
+//
+
+import UIKit
+
+class MainScreen: UIViewController, UITableViewDelegate, UITableViewDataSource  {
+   
+    
+    
+    var data = [String()]
```

Comment:
и по поводу приватности: давай пока руководствоваться правилом: по умолчанию делаем всё приватным, если потребуется изменять из вне - сделаешь публичным.
синтаксис приватных таков:
```suggestion
    private var _data = [String()]
```
"везде" исправь для свойств/функций